config/locales/client.en.yml

```diff
@@ -26,6 +25,11 @@ en:
       claim:
         title: "claim"
         help: "Assign topic to yourself"
+      reminders_frequency:
```

Comment:
I'm not very sure here but is this being used on the client side?

Reply:
It's being used by `RemindAssignsFrequencySiteSettings#frequency_for`